### ISSUE TITLE
.bdf format format issues 

### ISSUE THREAD

**Issue by TakWolf** [quiple/galmuri#38]
Galmuri9.bdf

```
STARTFONT 2.1
FONT -Galmuri9-Regular-R-Regular--12-12-75-75-c-80-iso10646-1
SIZE 12 75 75
FONTBOUNDINGBOX 17 17 -2 -3
STARTPROPERTIES 10      <----------------------
FAMILY_NAME Galmuri9
WEIGHT_NAME Regular
FONT_VERSION 2.376
COPYRIGHT Copyright (c) 2019-2023 Minseo Lee (itoupluk427@gmail.com)
FOUNDRY Made with Bits'n'Picas by Kreative Software
FONT_ASCENT 11
FONT_DESCENT 1
POINT_SIZE 10
X_HEIGHT 6
CAP_HEIGHT 9
ENDPROPERTIES          <----------------------
CHARS 20704
STARTCHAR U+0020
ENCODING 32
SWIDTH 235 0
DWIDTH 4 0
BBX 0 0 0 0
BITMAP
ENDCHAR
STARTCHAR U+0021
ENCODING 33
SWIDTH 235 0
DWIDTH 4 0
BBX 1 9 1 0
```

Properties (Between 'STARTPROPERTIES' and 'ENDPROPERTIES') value should be a int or a string (with double-quote) .
As the specification:  https://www.x.org/docs/BDF/bdf.pdf

```
Then come p lines consisting of a word for the property name followed by either an integer or string
surrounded by double-quote (octal 042). Internal double-quote characters are indicated by using two
in a row.
```

**Comment by TakWolf** [comment]
raise to: https://github.com/kreativekorp/bitsnpicas/issues/39

**Comment by TakWolf** [comment]
https://github.com/kreativekorp/bitsnpicas/pull/41

It seems to have been fixed.

**Comment by quiple** [comment]
> [kreativekorp/bitsnpicas#41](https://github.com/kreativekorp/bitsnpicas/pull/41)
> 
> It seems to have been fixed.

Thanks! I'll be releasing a new version soon.

**Comment by quiple** [comment]
Fixed in v2.38.8.
